HermaFx.DataAnnotations/MinElementsAttribute.cs

```diff
@@ -11,12 +11,12 @@ public class MinElementsAttribute : ValidationAttribute
 	{
 		private const string _defaultErrorMessage = "There should be at least {0} elements.";
 
-		private uint _elements;
+		public uint Elements;
```

Comment:
Field should not have a public visibility. Why did you change it in first place?